Misha/OB-365_erat_sieve/utest_sieve_of_erat.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+from main import sieve_of_eratosthenes
+
+
+class EratSieveTest(unittest.TestCase):
+
+    def test_erat_sieve_with_right_parameter(self):
+        primes = sieve_of_eratosthenes(6)
+        expected_result = [2, 3, 5]
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_border_parameter(self):
+        primes = sieve_of_eratosthenes(5)
+        expected_result = [2, 3, 5]
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_minimal_parameter(self):
+        primes = sieve_of_eratosthenes(2)
+        expected_result = [2]
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_non_type_object(self):
+        with self.assertRaises(TypeError) as context:
+            self.assertRaises(TypeError, sieve_of_eratosthenes(None))
```

Comment:
You have double check here. Remove one

Reply:
Same for other occurrence

Reply:
with self.assertRaises(SomeException) as cm:
                   do_something()
               the_exception = cm.exception
               self.assertEqual(the_exception.error_code, 3)

its an example from ctrl+clisk (case.py)
and i saw same examples in the internet. it is checking, if same error was raised